src/Microsoft.AspNetCore.Build/targets/Package.targets

```diff
@@ -1,18 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <!-- Put targets attached to the 'Clean' phase of the Standard Lifecycle here. -->
+  <!-- Put targets attached to the 'Clean' phase of the Standard Lifecycle here. -->
 
-    <Target Name="CleanPreviousPackOutput">
```

Comment:
Immediately: It was running after compile, which was putting stuff in that directory. I'll look at where to move it. Probably just to "Prepare" (the "Clean" targets are for cleaning the entire repo, which would destroy incremental builds, but cleaning `artifacts` in "Prepare" wouldn't do that because we still build in-place, i.e. in  `bin`/`obj`)